test/build/config-clean/package.json

```diff
@@ -8,7 +8,7 @@
   },
   "snowpack": {
     "buildOptions": {
-      "clean": false,
+      "clean": true,
```

Comment:
updated this test to test the non-default behavior

Reply:
If the `artifact.txt` is deleted from the test, how do we know it’s working?